src/Code.gs

```diff
@@ -614,7 +616,7 @@ FirebaseApp_._sendAllRequests = function (finalRequests, originalsRequests, db,
       continue;
     }
     
-    // All other cases are errors that we do not retry
+    // All other cases are errors on which we perform a retry
```

Comment:
@JeanRemiDelteil Can you confirm that this comment was not accurate?

Reply:
no, the comment was accurate, see my other comment